aws_cloud_cost/main.yaml

```diff
@@ -0,0 +1,164 @@
+# version: 0.0.1
+AWSTemplateFormatVersion: 2010-09-09
+Description: Datadog Cloud Cost Management Setup
+Parameters:
+  DDCloudCostBucketName:
+    Type: String
+    Description: The name of the S3 bucket for storing the cost and usage reports.
+    MinLength: 1
+  DDCloudCostBucketRegion:
+    Type: String
+    Description: The AWS region where the S3 bucket is located.
+    MinLength: 1
+  DDCloudCostBucketPrefix:
+    Type: String
+    Description: The prefix to be added to the cost and usage report files in the S3 bucket.
+    MinLength: 1
+  DDCloudCostReportName:
```

Comment:
Does it make sense to prefix these all with "DD"? I think it's kinda implied their Datadog related.